### DISCUSSION TITLE
Waht is stable version?

### DISCUSSION THREAD

**Discussion by Hoohaha** [celery/py-amqp#394] [Q&A]
Hi,

Today I found a serious issue in our production environment. There are large rabbitmq connections, which caused the server CPU usage is very high. Finnaly I found that the py-amqp not closing connections when celery worker exited. 

Checked with py-amqp releases on github. found it is fixed by https://github.com/celery/py-amqp/pull/374

So what version should I use for stable consideration?

**Comment by auvipy** [comment]
you can easily check the commit history and release notes to understand that, use the latest pypi release. you have questions ask them in forum first.
